### PR TITLE
added basic order test

### PR DESCRIPTION
**Pull request by andrei-gubarev**

### CODE REVIEW

**Comment by andrei-gubarev** [comment]
Hello.
My name is Andrey. I saw your job offer on upwork. 
my profile: https://www.upwork.com/freelancers/~01b1b655d2b43ea78c

I commit example order test. 
I'm sorry, if some goes wrong. I'm new with github.

I have created test which fill order form fields and check the existence of login form after submit button click. I did so because I cannot to regiser (see screenshot).

I hope it is what you need. If you need to show more skills and experience I would be glad to show you.
My linkedin profile: https://ru.linkedin.com/in/andrey-gubarev-2b587263
Thanks

**Comment by leonardopires** [comment]
Hello Andrey,

I really liked your proactivity. Thanks for pointing out that our testing environment is broken. I haven't noticed that until your pull request.

Your code is clean and I got really impressed how fast you did it. I'm still waiting for more submissions to make any decision, and I need to fix this issue you've found, so I'll let you know when the site is up. 

Since I don't know what the others will submit after the sign up screen is working, I would do some more tests when it gets up again. :)

We'll talk soon

File: src/WeedSeeker.Web.Test.Functional/WeedSeeker.Web.Test.Functional.csproj [hunk 1]
```diff
@@ -30,10 +30,26 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="nunit.core, Version=2.6.4.14350, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">
```
Comment (leonardopires) [inline src/WeedSeeker.Web.Test.Functional/WeedSeeker.Web.Test.Functional.csproj:33]: What was the reasoning behind chagning the NUnit version to from 3.01 to 2.6.4? Was it necessary to run in VS 2013 UI?

File: src/WeedSeeker.Web.Test.Functional/BaseTest.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+using System.Linq;
+using System.Text;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using WeedSeeker.Web.Test.Functional.Properties;
+
+namespace WeedSeeker.Web.Test.Functional
+{
+    public class BaseTest
```
Comment (leonardopires) [inline src/WeedSeeker.Web.Test.Functional/BaseTest.cs:12]: Good move. This was what I was expecting to see from an experienced developer.

File: src/WeedSeeker.Web.Test.Functional/WeedSeeker.Web.Test.Functional.csproj (hunk 1 above)
Comment (leonardopires) [inline src/WeedSeeker.Web.Test.Functional/WeedSeeker.Web.Test.Functional.csproj:33]: OK... I get it now. That's the package that makes VS2013 run nUnit 3.0 tests inside its own UI, just like VS 2015 does by default.

I just accepted your pull request. 

Congratulations. You made it to the master. :smile:

Comment (andrei-gubarev) [inline src/WeedSeeker.Web.Test.Functional/WeedSeeker.Web.Test.Functional.csproj:33]: No. Maybe it changed when I intalled NUnit Test Adapter earlier version (not for NUnit 3.0).
Also I have Installed NUnit 2.6 on my PC. Maybe it automattically changed when solution build
I have not noticed this change.

## 

С уважением,
Губарев Андрей

03.02.2016, 22:40, "Leonardo Pires" notifications@github.com:

File: src/WeedSeeker.Web.Test.Functional/BaseTest.cs (hunk 2 above)
Comment (andrei-gubarev) [inline src/WeedSeeker.Web.Test.Functional/BaseTest.cs:12]: Thanks. I am very pleased to hear that

**Comment by andrei-gubarev** [comment]
Hello.
I'm trying to register on http://alpha.weedseeker.net/.
After all fields filled I see message with error (see screenshot)
register_failed

**Comment by leonardopires** [comment]
What are you filling into the form?
Do you have that into an automated test I can run?

**Comment by andrei-gubarev** [comment]
I will do this test and create a pull request, ok?

**Comment by leonardopires** [comment]
Awesome!

I just sent you a message in upwork ;)

**Comment by andrei-gubarev** [comment]
I have answered your message :)

**Pull request merged**